Reference/google/protobuf/unittest_mset.pb.swift

```diff
@@ -304,7 +304,7 @@ struct ProtobufUnittest_RawMessageSet: SwiftProtobuf.Message {
 
 extension Proto2WireformatUnittest_TestMessageSet {
   var ProtobufUnittest_TestMessageSetExtension1_messageSetExtension: ProtobufUnittest_TestMessageSetExtension1 {
-    get {return getExtensionValue(ext: ProtobufUnittest_TestMessageSetExtension1.Extensions.message_set_extension)}
+    get {return getExtensionValue(ext: ProtobufUnittest_TestMessageSetExtension1.Extensions.message_set_extension) ?? ProtobufUnittest_TestMessageSetExtension1()}
```

Comment:
Apologies for being unresponsive.  Yes, this is fine.  (My question earlier was purely for clarification, not an objection.)